authors/apps/profiles/renderers.py

```diff
@@ -1,25 +1,6 @@
-import json
+from ..core.utils.general_renderer import GeneralRenderer
 
-from rest_framework.renderers import JSONRenderer
-from rest_framework.utils.serializer_helpers import ReturnList, ReturnDict
-
-
-class ProfileJSONRenderer(JSONRenderer):
 
+class ProfileJSONRenderer(GeneralRenderer):
     charset = 'utf-8'
-
-    def render(self, data, media_type=None, renderer_context=None):
```

Comment:
Thanks for saving us from repeated code. Great work harmonizing the JSON responses